float64 	preferred_pitch
float64[3] 	des_position
float64[3] 	des_normal
---
float64[5] 	joint_angles
bool 		feasible
bool 		arm_to_front
bool 		arm_bended_up
bool 		gripper_downwards